BondageClub/Screens/Inventory/ItemHands/SpankingToys/SpankingToy.js

```diff
@@ -169,6 +169,11 @@ const InventoryItemHandsSpankingToysOptions = [
 		Name: "Broom",
 		Property: { Type: "Broom" },
 		ExpressionTrigger: [{ Group: "Blush", Name: "Low", Timer: 5 }, { Group: "Eyebrows", Name: "Harsh", Timer: 5 }]
+	},{
+		Name: "Phone1",
+		Property: { Type: "Phone1" },
+		ExpressionTrigger: [{ Group: "Blush", Name: "Low", Timer: 10 }, { Group: "Eyebrows", Name: "Soft", Timer: 10 }, { Group: "Mouth", Name: "Frown", Timer: 10 }]
```

Comment:
are expression triggers needed? might be a bit strange to blush when picking up a phone

Reply:
Bit strange to blush when picking up a towel or a sock, too. Didn't know quite what was safe to remove so with the rest still having them I just left them in.

Reply:
yeah i agree, would be good to remove those from the items that dont make sense at some point.
You can accept this suggested edit if you want it, otherwise you can dismiss it
```suggestion
		ExpressionTrigger: []
```